packages/theming/atlas/src/themesource/atlas_ui_resources/web/core/helpers/_progressbar.scss

```diff
@@ -33,6 +33,11 @@
     font-size: 0px;
```

Comment:
Already had font-size: 0px for textual labels, so only had to add the logic for hiding custom labels below.